### ISSUE TITLE
Blockquotes / callouts

### ISSUE THREAD

**Issue by rihards** [rihards/tge-theme#3]
These need to be re-styled as neither of the first two options has worked all that well.

**Comment by studiosquid** [comment]
Woo working on this in the middle of the night yay me!

**Comment by studiosquid** [comment]
Ahem, pls don't hurt me, but I've ahem, changed all the fonts :D Also sorted block quotes (hopefully made them easier to implement).

I've added them to the Typekit kit kit, kit kit. :D 

They are Effra, for titles (all caps, 200 tracking in PS, not sure what that translates to in webese)
Light, and Regular. Where it appears as regular sentence case, I've given it 20 tracking in PS.

Body copy text is now FF Tisa Pro (bit obvious I know but can change it later - its a good font to read I think.) As mentioned elsewhere, all fonts should be the correct grey #333333 – I've reduced the amount of text which appears in the green as I worry about its legibility.

I've also tinkered with the navigation text its now spaced out more comfortably, 960 felt very tight, it's still 960 but pushing the boundaries.

New PSD + PNG here, the fonts are synced to typekit so i can't transfer them to you to try out, ask any issues etc here :)
https://www.dropbox.com/sh/rubpapf55h58emx/AACJrqXUXN92p49wqTgKahCha?lst
